scripts/zkai_attention_kv_two_head_fused_softmax_table_native_gate.py

```
@@ -0,0 +1,1003 @@
+        source_envelope_bytes=source_raw,
+        sidecar_envelope_bytes=sidecar_raw,
+    )
+    validate_fused_envelope(fused_envelope, source_input, run_native=False)
+    verify_fused_envelope_bytes_with_native_cli(fused_raw, str(FUSED_ENVELOPE_JSON))
+
+    mutation_results = []
+    for name, mutated, run_native in mutation_cases(fused_envelope):
+        try:
+            validate_fused_envelope(mutated, source_input, run_native=run_native)
+        except Exception as err:  # noqa: BLE001 - gate records exact rejection surface.
+            mutation_results.append({"name": name, "rejected": True, "error": str(err)})
```

Ruff (0.15.12)

[warning] `try`-`except` within a loop incurs performance overhead

(PERF203)